packages/apps/graph/tsconfig.json

```diff
@@ -2,6 +2,9 @@
   "extends": "./node_modules/@kadena-dev/shared-config/tsconfig-base.json",
   "compilerOptions": {
     "types": [".kadena/pactjs-generated", "node"],
-    "target": "ES2020"
+    "target": "ES2020",
+    "paths": {
+      "@src/*": ["./src/*"]
+    }
```

Comment:
Would it be nicer to have a path for each directory in `./src/`?

Reply:
Yep, done!